content/webapp/pages/search/index.tsx

```diff
@@ -842,137 +618,39 @@ export const getServerSideProps: GetServerSideProps<
-      const storiesResults = await getArticles({
-        params: {
-          ...query,
-          sort: getQueryPropertyValue(query.sort),
-          sortOrder: getQueryPropertyValue(query.sortOrder),
-        },
-        pageSize: 4,
-        toggles: serverData.toggles,
-      });
-
-      if (storiesResults.type === 'Error') contentQueryFailed = true;
 
-      stories = getQueryResults({
-        categoryName: 'stories',
-        queryResults: storiesResults as
-          | ContentResultsList<Article>
-          | WellcomeApiError,
-      });
-
-      // Events
-      const eventsResults = await getEvents({
-        params: {
-          ...query,
-          sort: getQueryPropertyValue(query.sort),
-          sortOrder: getQueryPropertyValue(query.sortOrder),
-        },
-        pageSize: 3,
-        toggles: serverData.toggles,
-      });
-
-      if (eventsResults.type === 'Error') contentQueryFailed = true;
-
-      events = getQueryResults({
-        categoryName: 'events',
-        queryResults: eventsResults as
-          | ContentResultsList<EventDocument>
-          | WellcomeApiError,
-      });
-
-      // Works
-      const worksResults = await getWorks({
-        params,
-        pageSize: 5,
-        toggles: serverData.toggles,
-      });
-      works = getQueryResults({
-        categoryName: 'works',
-        queryResults: worksResults,
-      });
-
-      // Images
-      const imagesResults = await getImages({
-        params,
-        pageSize: 10,
-        toggles: serverData.toggles,
-      });
-      images = getQueryResults({
-        categoryName: 'images',
-        queryResults: imagesResults,
-      });
+    // All/Addressables
+    const contentResults = await getAddressables({
+      params: {
+        ...query,
+      },
+      pageSize: 20,
+      toggles: serverData.toggles,
+    });
 
-      // If all three queries fail, return an error page
-      if (
-        imagesResults.type === 'Error' &&
-        worksResults.type === 'Error' &&
-        contentQueryFailed
-      ) {
-        // Use the error from the works API as it is the most mature of the 3
-        return appError(
-          context,
-          worksResults.httpStatus,
-          worksResults.description || worksResults.label
-        );
-      }
+    if (contentResults.type === 'Error') {
+      contentQueryFailed = true;
+      console.error(contentResults.label + ': Error fetching addressables');
     }
 
-    const apiToolbarLinks = serverData.toggles.allSearch.value
-      ? [
-          contentResults && contentResults.type !== 'Error'
-            ? {
-                id: 'content-api',
-                label: 'Content API query',
-                link: contentResults._requestUrl,
-              }
-            : undefined,
-        ].filter(isNotUndefined)
-      : [];
+    const apiToolbarLinks = [
+      contentResults && contentResults.type !== 'Error'
```

Comment:
np (these aren't due to your changes, I'm just reading through the file and spotting nitpicky improvements, feel free to ignore)
```suggestion
      contentResults && !contentQueryFailed
```

Reply:
TypeScript doesn't like it  ¯\\\_(ツ)\_/¯

Reply:
Oh who knows